Lykke.Service.LykkeService/src/Lykke.Service.LykkeService/Properties/launchSettings.json

```diff
@@ -3,15 +3,15 @@
     "LykkeService local dev settings": {
       "commandName": "Project",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "Development",
-        "SettingsUrl": "token_to_dev_settings"
+        "SettingsUrl": "appsettings.Development.json",
```

Comment:
Let's avoid any appsettings.*.json files here - use launchSettings.json for this

Reply:
@starkmsu, What do you mean here? We still could you local settings file. I believe appsettings.Development.json is suitable name for local settings file.